File: THI_CUM_LAN_2/contestants/nguyen_duy_hung/BSET.cpp
#include <bits/stdc++.h>
using namespace std;
int main()
{
    ifstream inp("BSET.INP");
    ofstream out("BSET.OUT");
    int a,b;
    inp >> a >> b;
    vector<int> A;
    vector<int> B;
    while(a--)
    {
        int num;
        inp >> num;
        A.push_back(num);
    }
    while(b--)
    {
        int num1;
        inp >> num1;
        B.push_back(num1);
    }
    int k=0;
    for(int i=0;i<B.size();i++)
        {for(int j=0;j<A.size();j++)
            {if(A[j]==B[i])
                k++;}}
    out << A.size() + B.size() - k << " " << k;
    out.close();
    return 0;
}

File: THI_CUM_LAN_2/contestants/190011/bset.py
with open("bset.inp","r") as fi:
    a=list(map(int,fi.readline().split()))
    b=list(map(int,fi.readline().split()))
with open("bset.out","w") as fo:
    x=list(set(a) & set(b))
    y=list(set(a) | set(b))
    for i in y:
        fo.write(str(i)+" ")
    fo.write(""+"\n")
    for i in x:
        fo.write(str(i)+" ")
fi.close()
fo.close()